internal/framework/modifiers.go

```diff
@@ -3,6 +3,7 @@
 // This software is licensed under a BSD 3-Clause License.
 // See the LICENSE file provided with this software.
 
+//nolint:forcetypeassert // Type checker cannot track type guarantees in comments
```

Comment:
Are there so many lint errors in this file so that we exclude it?

Reply:
```
internal/framework/modifiers.go:134:2: type assertion must be checked (forcetypeassert)
	resp.PlanValue = genericResponse.PlanValue.(types.Bool)
	^
internal/framework/modifiers.go:143:2: type assertion must be checked (forcetypeassert)
	resp.PlanValue = genericResponse.PlanValue.(types.Int64)
	^
internal/framework/modifiers.go:152:2: type assertion must be checked (forcetypeassert)
	resp.PlanValue = genericResponse.PlanValue.(types.Float64)
	^
internal/framework/modifiers.go:161:2: type assertion must be checked (forcetypeassert)
	resp.PlanValue = genericResponse.PlanValue.(types.Number)
	^
internal/framework/modifiers.go:170:2: type assertion must be checked (forcetypeassert)
	resp.PlanValue = genericResponse.PlanValue.(types.String)
	^
internal/framework/modifiers.go:179:2: type assertion must be checked (forcetypeassert)
	resp.PlanValue = genericResponse.PlanValue.(types.List)
	^
internal/framework/modifiers.go:188:2: type assertion must be checked (forcetypeassert)
	resp.PlanValue = genericResponse.PlanValue.(types.Map)
	^
internal/framework/modifiers.go:197:2: type assertion must be checked (forcetypeassert)
	resp.PlanValue = genericResponse.PlanValue.(types.Object)
```